components/aws/sagemaker/test/integration/component_tests/test_train.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import os
+import utils
+from utils import kfp_client_utils
+from utils import argo_utils
+from utils import minio_utils
+from utils import sagemaker_utils
+
+
+def test_simple_mnist_training(
+    kfp_client, experiment_id, s3_client, sagemaker_client, region
+):
+
+    pipeline_name = "simple_mnist_training"
+    test_file_dir = os.path.join("resources/", pipeline_name)
+    run_id, status = kfp_client_utils.compile_run_monitor_pipeline(
+        kfp_client, experiment_id, test_file_dir, pipeline_name, 1200
```

Comment:
Looks like the training job itself has a timeout of 3600 (assuming units are same), shouldn't this timeout be at least that much?

Reply:
ack